src/main/kotlin/com/vk/admstorm/git/GitUtils.kt

```diff
@@ -562,6 +562,19 @@ object GitUtils {
         }
     }
 
+
+    private const val getUserNameCommand = "git -c core.quotepath=false config --local --null --get user.name"
```

Comment:
Please add here comment like the above other constants

Reply:
fixed